pkgs/press/src/defaults.ts

```diff
@@ -1,17 +1,30 @@
 import { git, execSync, forceExit } from '@percolate/cli-utils'
 
 export function getBranch() {
+    if (process.env.GITHUB_REF) {
+        // GITHUB_REF=refs/heads/branch_name
+        const ref = process.env.GITHUB_REF
+        const match = ref.match(/refs\/heads\/(.*)$/)
+        if (!match) return forceExit(`Unable to determine branch name in ${ref}`)
+        return match[1]
+    }
     return process.env.CIRCLE_BRANCH || git.getCurrBranch()
 }
 
 export function getHash() {
+    if (process.env.GITHUB_SHA) {
```

Comment:
should check for `process.env.GITHUB_ACTIONS` presence, so it is clear in which cases we should have `GITHUB_SHA` / `GITHUB_REPOSITORY` / `GITHUB_REF` ?

Reply:
https://github.com/percolate/blend/pull/66/commits/8ae6afda072102fa68e3df654d62e3dc724449c6